src/views/components/address/Address.js

```diff
@@ -9,7 +9,10 @@ const Address = ({
   options,
   ...restProps
 }) => (
-  <Text {...restProps}>
+  <Text
+    {...restProps}
+    testID="address"
```

Comment:
I believe we should allow `testID` to be overwritable - if we remove `testID="address"` and replace it with `testID={testID}`, and where we destructure the props for this component add `testID = 'address'`, it would make it much more flexible in our tests.

Reply:
I just realised that in this case we can simply move `testID="address"` one line up and achieve the same thing, but this method is interchangeable with what I said above for the rest of the components.

Reply:
Yeah agreed. Always should be a prop on each component with a default.